src/main/java/net/obvj/performetrics/util/print/PrintStyleBuilder.java

```diff
@@ -0,0 +1,205 @@
+package net.obvj.performetrics.util.print;
+
+import net.obvj.performetrics.util.DurationFormat;
+import net.obvj.performetrics.util.DurationFormatter;
+
+public class PrintStyleBuilder
+{
+    public static final String DEFAULT_FORMAT = "%s";
+
+    private DurationFormat durationFormat;
+    private boolean printLegend;
+    private boolean printHeader;
+    private boolean printSectionSummary;
+    private String headerFormat;
+    private String rowFormat;
+    private String sectionHeaderFormat;
+    private String sectionSummaryRowFormat;
+    private String simpleLine;
+    private String alternativeLine;
```

Comment:
Try to work with two interfaces so one trying to build a style for the summarized format doesn't need to deal with unused properties.

Reply:
Too much complexity for hiding only three methods.